manifests/i/IObit/MalwareFighter/9.3.0/IObit.MalwareFighter.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: IObit.MalwareFighter
+PackageVersion: 9.3.0

```

```suggestion
PackageVersion: 9.3.0.744
```
This must match Control Panel.

You'll also need to change `manifests/i/IObit/MalwareFighter/9.3.0` to `manifests/i/IObit/MalwareFighter/9.3.0.744`